.github/workflows/build-extensions.yml

```diff
@@ -186,6 +192,7 @@ jobs:
           do
             echo "Testing ${plugin_array[$i]} :"
             cd ${test_prefix}/${plugin_array[$i]}
+            # Tests of wasm_bpf require root privilege
```

Comment:
Why do we have this comment here without any command called?
Should this be with Line 162?

Reply:
Moved to proper position now